src/core/symbology-ng/qgssymbollayerv2.cpp

```diff
@@ -225,6 +225,8 @@ QgsMarkerSymbolLayerV2::QgsMarkerSymbolLayerV2( bool locked )
 QgsLineSymbolLayerV2::QgsLineSymbolLayerV2( bool locked )
     : QgsSymbolLayerV2( QgsSymbolV2::Line, locked )
     , mWidthUnit( QgsSymbolV2::MM )
+    , mOffset( 0 )
+    , mOffsetUnit( QgsSymbolV2::MM )
 {
```

Comment:
@nyalldawson I guess that in this case the default ctor (i.e. QgsLineSymbolLayerV2( bool locked = false )) is called. Since locked is not in the ctor arguments in either, I can't se a bug there.